Makefile

```diff
@@ -46,7 +48,7 @@ $(BIN)/golangci-lint: | $(BIN) ; $(info $(M) getting golangci-lint $(GOLANGCI_VE
 
 .PHONY: clean-cluster
 clean-cluster: | $(KO) $(KUSTOMIZE) clean-cr; $(info $(M) clean $(TARGET)…) @ ## Cleanup cluster
-	-$(KUSTOMIZE) build config/$(TARGET) | $(KO) delete -f -
+	-$(KUSTOMIZE) build --load-restrictor LoadRestrictionsNone config/$(TARGET) | $(KO) delete -f -
```

Comment:
What does `--load-restrictor LoadRestrictionsNone` and why is it needed ?

Reply:
in latest version of `kustomize` we cannot bring in resources which are outside base root.
In our case, the kustomization for bundle generation is defined in `operatorhub/<platform>/kustomization.yaml` . This pulls resources from:

- `operatorhub/<platform>/base`
- `operatorhub/<platform>/overlay/<strategy-name>`
to get csv stub and
- `../../config/<platform-name>` (to get all necessary resources)

the `../../config/` will not work with kustomize if we don't have that flag.

Reply:
The need for all this is to make the resource accumulation for CSV generation as close to how it is done in an Operator-sdk project.

eg: 
- input collection is specified through this: https://github.com/operator-framework/operator-sdk/tree/master/testdata/go/v3/memcached-operator/config/manifests
- so that the bundle will be generated here:https://github.com/operator-framework/operator-sdk/tree/master/testdata/go/v3/memcached-operator/bundle

Reply:
I see. Can we have a small comment about this ? 😛

Reply:
sure 😝 👍